# removes ugliness of popup menu of icon from task bar on Windows
# see http://www.mail-archive.com/ekiga-devel-list@gnome.org/msg03181.html and http://stackoverflow.com/questions/1138891/gtk-statusicon-and-gtk-menu-on-windows
# it is not integrated in ekiga because this bug will probably be fixed
#   by future GTK+ releases

diff -ur ekiga.orig/lib/engine/gui/gtk-frontend/statusicon.cpp ekiga/lib/engine/gui/gtk-frontend/statusicon.cpp
--- ekiga.orig/lib/engine/gui/gtk-frontend/statusicon.cpp	2009-08-31 14:04:57.000000000 +0200
+++ ekiga/lib/engine/gui/gtk-frontend/statusicon.cpp	2009-08-31 14:05:17.000000000 +0200
@@ -257,7 +257,11 @@
 
   gtk_menu_popup (GTK_MENU (popup),
                   NULL, NULL,
+#ifdef WIN32
+                  NULL, NULL,
+#else
                   (GtkMenuPositionFunc)gtk_status_icon_position_menu, icon,
+#endif
                   button, activate_time);
 }
 
